packages/block-library/src/image/utils.js

```diff
@@ -11,21 +6,19 @@ import { NEW_TAB_REL } from './constants';
 export function removeNewTabRel( currentRel ) {
 	let newRel = currentRel;
 
-	if ( currentRel !== undefined && ! isEmpty( newRel ) ) {
-		if ( ! isEmpty( newRel ) ) {
-			NEW_TAB_REL.forEach( ( relVal ) => {
-				const regExp = new RegExp( '\\b' + relVal + '\\b', 'gi' );
-				newRel = newRel.replace( regExp, '' );
-			} );
+	if ( currentRel !== undefined && newRel ) {
```

Comment:
The changeset of this file might look huge, so to help with that you can view the PR with hidden whitespace:

![Screenshot 2023-05-12 at 16 41 13](https://github.com/WordPress/gutenberg/assets/8436925/df929943-69b0-4185-ba0d-5712eb8f7749)

What we're doing here is we're removing a duplicated `! isEmpty( newRel )` condition which already exists in the top-level `if`.